LeaderboardBackend/Controllers/JudgementsController.cs

```diff
@@ -0,0 +1,94 @@
+using LeaderboardBackend.Authorization;
+using LeaderboardBackend.Controllers.Annotations;
+using LeaderboardBackend.Models.Entities;
+using LeaderboardBackend.Models.Requests;
+using LeaderboardBackend.Services;
+using LeaderboardBackend.ViewModels;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LeaderboardBackend.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class JudgementsController : ControllerBase
+{
+	private readonly ILogger _logger;
+	private readonly IJudgementService _judgementService;
+	private readonly IRunService _runService;
+	private readonly IUserService _userService;
+
+	public JudgementsController(
+		ILogger<JudgementsController> logger,
+		IJudgementService judgementService,
+		IRunService runService,
+		IUserService userService
+	)
+	{
+		_logger = logger;
+		_judgementService = judgementService;
+		_runService = runService;
+		_userService = userService;
+	}
+
+	/// <summary>Gets a Judgement from its ID.</summary>
+	/// <response code="200">The Judgement with the provided ID.</response>
+	/// <response code="404">If no Judgement can be found.</response>
+	[ApiConventionMethod(typeof(Conventions),
+						 nameof(Conventions.Get))]
+	[AllowAnonymous]
+	[HttpGet("{id}")]
+	public async Task<ActionResult<JudgementViewModel>> GetJudgement(long id)
+	{
+		Judgement? judgement = await _judgementService.GetJudgement(id);
+		if (judgement is null)
+		{
+			return NotFound();
+		}
+
+		return Ok(new JudgementViewModel(judgement));
+	}
+
+	/// <summary>Creates a judgement for a run.</summary>
+	/// <response code="201">The created judgement.</response>
+	/// <response code="400">The request body is malformed.</response>
+	/// <response code="403">The run has pending participations.</response>
+	/// <response code="404">For an invalid judgement.</response>
+	[ApiConventionMethod(typeof(Conventions),
+						nameof(Conventions.Post))]
+	[ProducesResponseType(StatusCodes.Status403Forbidden)]
```

Comment:
This produces the warning mentioned below. Mixing `ApiConventionMethod` and `ProducesResponseType` is apparently not liked.

Reply:
So I had a suspicion, and I was correct.

You just need to have `ProducesResponseType` be above `ApiConventionMethod`. Lmao